cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStore.java

```diff
@@ -95,7 +95,12 @@ public void storeTrigger(OperableTrigger newTrigger, boolean replaceExisting) th
   public void storeJobsAndTriggers(Map<JobDetail, Set<? extends Trigger>> triggersAndJobs,
                                    boolean replace) throws JobPersistenceException {
     super.storeJobsAndTriggers(triggersAndJobs, replace);
```

Comment:
Yes. Its a different method call on super not at all related to storeJob and storeTrigger.